### PR TITLE
passing tests to get all books and one book by id and rendering books…

### PR DESCRIPTION
**Pull request by yaseenagag**
… via pug 💯

### CODE REVIEW

File: config/database.js [hunk 1]
```diff
@@ -89,17 +89,17 @@ const getBook = (id) => {
   WHERE books.id = ${id}`)
 }
 
-const getBooks = ({ page, title, author, year, cont}) => {
-  page = parsInt( page || 1 )
+const getBooks = ({ page, title, author, year, count}) => {
```
Comment (devonwesley) [inline config/database.js:92]: Call it ```const getBooksOffset10 = ...```

**Comment by devonwesley** [comment]
Not done reviewing, but the description say's "passing test to get all ....", I don't see the test files in this commit.

**Comment by devonwesley** [comment]
I pulled the repo and 14 of the test passed Yay! but 2 of them didn't below:

screen shot 2017-04-05 at 8 34 34 pm

And when i ran the test initially 8 test failed. I had to setup my database to get most of the test to work. 
There were no clear instruction on how to do this either. 
So I had to go into the code and find the DB name "bookworm", I like it, figure out if you had a "schema.sql" or some other way to create the database and then how to run the schema.sql file.
You want to be clear, to help out your fellow developers that's all, most of them or newer dev's.

Alot of good work in there Im going to finishing reviewing the PR in the AM. Goodjob!

File: config/database.js (hunk 1 above)
Comment (yaseenagag) [reply config/database.js:92]: noted and thanks dev,  I will make sure to ccreate a script that builds the database and injects the schema.sql => all the tables into our database.  Also i will take into consideration how to improve my naming conventions for my fellow teammates. 🥇

**Comment by yaseenagag** [comment]
I will write a script to take care of creating the database and the schema plus the books JSON to be injected using few npm commands.  All will be taken care of first thing in the morning.

**Pull request merged**